### DISCUSSION TITLE
ALTER TABLE: ability to create more than one column in a single query

### DISCUSSION THREAD

**Discussion by gdegentile1** [duckdb/duckdb#9689] [Ideas]
### What happens?

Hello,

I am currently evaluating DuckDB and I am already a big fan (I have previous experience with MonetDB which is unfortunately not very active).
I would like to create multiple columns in a single query.

**Example:**
`ALTER TABLE integers ADD COLUMN k INTEGER, n INTEGER;`

The reason is that I am processing OLAP cube Analytics which can rely on different in-memory databases (ex: H2, MonetDB, SQLITE, ...) and columns are created with a single query.
I have already adapted the code to create columns one by one in case the single query fails, but it would be nice to have this feature.

Thank you.

Best regards,
Guillaume

### To Reproduce

Create a table and try to add more than one column in a single query

**Example:**
`ALTER TABLE integers ADD COLUMN k INTEGER, n INTEGER;`

### OS:

Windows

### DuckDB Version:

0.9.1

### DuckDB Client:

Java

### Full Name:

Guillaume de GENTILE

### Affiliation:

FINBOX SOLUTIONS

### Have you tried this on the latest `main` branch?

I have tested with a release build (and could not test with a main build)

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] Yes, I have

**Comment by szarnyasg** [comment]
Hi Guillame, thanks for raising this. This is a feature request, so I converted it into a GitHub Discussion.
